test/UnitTimeoutNone.cpp

```diff
@@ -0,0 +1,286 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; fill-column: 100 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <config.h>
+
+#include <memory>
+#include <string>
+
+#include <HttpRequest.hpp>
+#include <Socket.hpp>
+
+#include <Poco/DOM/DOMParser.h>
```

Comment:
Safe to assume we don't need XML parsing for ping timeout tests.

Also, this test framework is not recommended for this use-case. See UnitWSD and family for event-based tests.

It is possible to use these tests for this use-case, but they are far harder to debug and fragile in the face of even ordering. Which is why we use UnitBase, UnitWSD, etc.

Reply:
didn't clean the test up indeed (includes, exception-catch), will do when finishing the test